### ISSUE TITLE
'content' not present at all in API Elements for parameter with no sample value

### ISSUE THREAD

**Issue by honzajavorek** [apiaryio/drafter#305]
```apib
FORMAT: 1A
# Beehive API
## Honey [/honey{?param}]
### Remove [DELETE]
+ Parameters
    + nonexisting (string, required)
+ Response
```

The parameter gets parsed as:

#### protagonist @ 1.3.0

```json
{
  "element": "member",
  "attributes": {
    "typeAttributes": [
      "required"
    ]
  },
  "content": {
    "key": {
      "element": "string",
      "attributes": {
        "sourceMap": [
          {
            "element": "sourceMap",
            "content": [
              [
                90,
                31
              ]
            ]
          }
        ]
      },
      "content": "nonexisting"
    },
    "value": {
      "element": "string"
    }
  }
}
```

#### protagonist @ 1.3.0-pre.1

```json
{
  "element": "member",
  "attributes": {
    "typeAttributes": [
      "required"
    ]
  },
  "content": {
    "key": {
      "element": "string",
      "attributes": {
        "sourceMap": [
          {
            "element": "sourceMap",
            "content": [
              [
                90,
                31
              ]
            ]
          }
        ]
      },
      "content": "nonexisting"
    },
    "value": {
      "element": "string",
      "content": ""
    }
  }
}
```

Notice the change:

```diff
 "value": {
   "element": "string",
-  "content": ""
 }
```

Is this correct behavior or is it a bug?

**Comment by honzajavorek** [comment]
The problem is that if there is no `content` property, even one of `""` or `null` value, I can't say whether the thing is refracted element or the final value. My algorithm doesn't see `content`, so it considers the `{"element": "string"}` part to be the final value of the `"value": {...` node.

**Comment by pksunkara** [comment]
This is intended because we fixed the bug in #286

**Comment by honzajavorek** [comment]
Makes sense. Took me a while to understand that Refract elements without content are completely valid part of Refract.
